lib/galaxy/jobs/__init__.py

```diff
@@ -2099,22 +2100,33 @@ def container_monitor_command(self, container, **kwds):
         from os import getcwd
         exec_dir = kwds.get('exec_dir', abspath(getcwd()))
         work_dir = self.working_directory
-        container_config = os.path.join(work_dir, "container_config.json")
-
-        # TODO: implement callback via URL callback for configuring ports instead
-        #       of fs polling.
-        # if self.app.config.galaxy_infrastructure_url_set:
-        #    # TODO: settable from job destination...
-        #    infrastructure_url = self.app.config.galaxy_infrastructure_url
-        # else:
-        #    infrastructure_url = "host.docker.internal"
+        configs_dir = os.path.join(work_dir, "configs")
```

Comment:
We also have `__build_config_files` in the ToolEvaluator class, maybe this logic could be split between the ToolEvaluator and the Interactive tool class ? 

Those imports be dropped btw, we already `import os` at the top.